src/Microsoft.Health.Dicom.SqlServer/Microsoft.Health.Dicom.SqlServer.csproj

```diff
@@ -75,8 +75,10 @@
     <TInitSqlScript Include="Features\Schema\Sql\Scripts\TransactionCheck.sql" />
   </ItemGroup>
 
-  <!-- Target contains sql build tasks -->
-  <Import Project="$(NuGetPackageRoot)\microsoft.health.tools.sql.tasks\$(HealthcareSharedPackageVersion)\build\Sql.targets" />
+  <!-- Execute target for generate SQL scripts -->
+  <!-- Condition skips outer build when PkgMicrosoft_Health_Tools_Sql_Tasks is not yet populated -->
+  <!-- TODO: Package should import its own target file if it matches the name of the package -->
+  <Import Condition="'$(TargetFramework)' != ''" Project="$(PkgMicrosoft_Health_Tools_Sql_Tasks)\build\Sql.targets" />
```

Comment:
We can use `GeneratePathProperty` to get an MSBuild property for the NuGet package instead of building the path ourselves based on the version